.github/workflows/publish.js.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     - run: npm install
       name: Install dependencies
     - run: rm -rf package-lock.json
+    - run: rm -rf node_modules/appium
```

Comment:
I'm not quite sure we should do this before running tests as they would most likely fail.
We should also probably be smart here and remove all dependencies that are listed as peer ones in package.json

Should the same change be applied to other drivers btw?